codalab/codalab/context_processors.py

```diff
@@ -21,4 +20,8 @@ def common_settings(request):
         'local_ace_editor': codalab_settings.LOCAL_ACE_EDITOR,
         'is_dev': codalab_settings.IS_DEV,
         'USE_AWS': codalab_settings.USE_AWS,
+
+        # Just get the base of the URL
+        'CHAHUB_URL': '//{}'.format(settings.CHAHUB_API_URL.split("//")[-1].split("/")[0].split('?')[0]),
```

Comment:
Should we have a function or something nicer format this?

Reply:
yes and that function should have tests! :)